chromium_src/third_party/blink/renderer/modules/quota/navigator_storagetest.cc

```diff
@@ -0,0 +1,58 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "base/path_service.h"
+#include "brave/browser/brave_content_browser_client.h"
+#include "brave/common/brave_paths.h"
+#include "chrome/browser/ui/browser.h"
+#include "chrome/common/chrome_content_client.h"
+#include "chrome/test/base/in_process_browser_test.h"
+#include "chrome/test/base/ui_test_utils.h"
+#include "content/public/test/browser_test.h"
+#include "content/public/test/browser_test_utils.h"
+
+const char kStorageEstimateTest[] = "/storage_estimate.html";
+
+class NavigatorStorageEstimateQuotaTest : public InProcessBrowserTest {
+ public:
+  void SetUpOnMainThread() override {
+    InProcessBrowserTest::SetUpOnMainThread();
+
+    content_client_.reset(new ChromeContentClient);
+    content::SetContentClient(content_client_.get());
+    browser_content_client_.reset(new BraveContentBrowserClient());
+    content::SetBrowserClientForTesting(browser_content_client_.get());
+    content::SetupCrossSiteRedirector(embedded_test_server());
```

Comment:
I am not sure if any of this is needed in all of these 3 navigator_* tests. And if not, then you won't need the dependency on //brave/browser. They seem to pass with all this removed. cc: @jumde

Reply:
Makes sense! Works for me as well, I updated all 3